backend/entityservice/views/project.py

```diff
@@ -294,7 +295,7 @@ def upload_json_clk_data(dp_id, clk_json, parent_span):
         )
 
     with opentracing.tracer.start_span('update-db', child_of=parent_span) as span:
```

Comment:
As you're removing some none-used parameter, `span` does not seem used here neither.
Also, the span which is started in named `update-db`, but line 248, there is another span named `update-database` (observed only because there was changes around these lines in this PR :) ). 
Should the naming be consistent?